src/v2/providers/firestore.ts

```diff
@@ -153,6 +177,50 @@ export function onDocumentWritten<Document extends string>(
   return onChangedOperation(writtenEventType, documentOrOpts, handler);
 }
 
+/**
+ * Event handler which triggers when a document is created, updated, or deleted in Firestore.
```

Comment:
This and most other instances of "which" in these comments is better as "that" (more restrictive, rather than additive). 

https://prowritingaid.com/art/438/-Which--or--That-%3A-Know-When-to-Use-Each.aspx